README.md

```diff
@@ -35,6 +35,9 @@ import { RequestError } from "@octokit/request-error";
 </tbody>
 </table>
 
+> [!IMPORTANT]
+> As we use [conditional exports](https://nodejs.org/api/packages.html#conditional-exports), you will need to adapt your `tsconfig.json`. See the TypeScript docs on [package.json "exports"](https://www.typescriptlang.org/docs/handbook/modules/reference.html#packagejson-exports).
```

Comment:
> you will need to adapt your `tsconfig.json`

I looked at [the docs you linked](https://www.typescriptlang.org/docs/handbook/modules/reference.html#packagejson-exports) but I'm not sure what changes I'll need to do in my `tsconfig.json`?

Reply:
sindresorhus's gist links to these doc sections for TypeScript,

https://www.typescriptlang.org/tsconfig#module
https://www.typescriptlang.org/docs/handbook/modules/reference.html#node16-nodenext

Reply:
Maybe mentioned in the note that `"moduleResolution": "node"` is no longer supported, and recommend what should be used instead, before linking to the docs?

Reply:
```suggestion
> As we use [conditional exports](https://nodejs.org/api/packages.html#conditional-exports), you will need to adapt your `tsconfig.json` by setting `"moduleResolution": "node16", "module": "node16"`.
>
> See the TypeScript docs on [package.json "exports"](https://www.typescriptlang.org/docs/handbook/modules/reference.html#packagejson-exports).<br>
> See this [helpful guide on transitioning to ESM](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) from [@sindresorhus](https://github.com/sindresorhus)
```

What do you think of this

Reply:
I think that's great! 

At first I was confused why node16 as Node 16 is already out of maintenance, but it seems the best option based on  https://www.typescriptlang.org/docs/handbook/modules/reference.html#packagejson-exports

Reply:
That's just the name of the modules resolution algorithm, :man_shrugging: It doesn't really have anything to do with Node 16. I guess it was the LTS release of Node when TS 4.7 was released